extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/HostAliasConfig.java

```diff
@@ -0,0 +1,28 @@
+
+package io.quarkus.kubernetes.deployment;
+
+import java.util.Optional;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class HostAliasConfig {
+
+    /**
+     * The ip address
+     *
+     * @return The ip.
+     */
+    @ConfigItem
+    Optional<String> ip;
+
+    /**
+     * The hostnames (comma separated) to resolve to the ip
+     *
+     * @return The path.
+     */
+    @ConfigItem
+    Optional<String> hostnames;
```

Comment:
If this is going to contain multiple values, why not make it a List?

Reply:
True... 
Done